index.org

```diff
@@ -585,6 +585,11 @@ version:
 11.7.4
 #+end_src
 
+#+name: reagent-flow-version
+#+begin_src text
+11.7.5
```

Comment:
This can be confusing, what about later reactflow upgrades to 11.7.5? Reagent-flow 11.7.5 would still be based on reactflow version 11.7.4.

Reply:
We can probably postfix custom identifier+numbers after 11.7.4?

Reply:
Done, changed it so reagent-flow's version is 1.7.4-clj.01. Changes in the wrapper itself could be released by just bumping the postfix version, i.e, 1.7.4-dev.02, 1.7.4-dev.03